cmd/nodecmd/validateSubnet.go

```diff
@@ -0,0 +1,181 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package nodecmd
+
+import (
+	"encoding/json"
+	"errors"
+	"io"
+	"os"
+	"time"
+
+	"github.com/ava-labs/avalanchego/vms/platformvm/status"
+
+	"github.com/ava-labs/avalanche-cli/pkg/ansible"
+
+	"github.com/ava-labs/avalanche-cli/cmd/subnetcmd"
+	"github.com/ava-labs/avalanche-cli/pkg/models"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/spf13/cobra"
+)
+
+var (
+	deployTestnet   bool
+	deployMainnet   bool
+	keyName         string
+	subnetName      string
+	useLedger       bool
+	ledgerAddresses []string
+	weight          uint64
+	duration        time.Duration
+
+	ErrMutuallyExlusiveKeyLedger = errors.New("--key and --ledger,--ledger-addrs are mutually exclusive")
+	ErrStoredKeyOnMainnet        = errors.New("--key is not available for mainnet operations")
+	ErrNoBlockchainID            = errors.New("failed to find the blockchain ID for this subnet, has it been deployed/created on this network?")
+)
+
+func newValidateSubnetCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "validateSubnet [clusterName]",
+		Short: "(ALPHA Warning) Join a subnet as a validator",
+		Long: `(ALPHA Warning) This command is currently in experimental mode.
+
+The node validateSubnet command enables all nodes in a cluster to be validators of a Subnet.
+If The command is run before the nodes are Primary Network validators, the command will first
```

Comment:
capitalization `The`

Reply:
would it make more sense to fail if you're not already a primary network validator? Genuine question, not a binding suggestion